src/theme.cpp

```diff
@@ -164,28 +175,19 @@ void Theme::setCurrentTheme(const QString& themeName) {
 void Theme::setUsingSystemTheme(const bool usingSystemTheme) {
   SettingsHolder::instance()->setUsingSystemTheme(usingSystemTheme);
 
-#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
   if (usingSystemTheme) {
     setToSystemTheme();
   }
-#endif
 }
 
-#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
 void Theme::setToSystemTheme() {
   if (!Feature::get(Feature::Feature_themeSelection)->isSupported()) {
```

Comment:
```suggestion
  if (!Feature::get(Feature::Feature_themeSelection)->isSupported() || !Feature::get(Feature::Feature_themeSelectionIncludesAutomatic)->isSupported()) {
```

Since we're removing the Qt version check, we need another safety check here so we don't try to update this. I know we have the safety check down below in `currentSystemTheme`, but without this early return this can still lead to weird corner cases - for instance, if you turn the theme selection feature on, set it to "Dark", then turn the feature back off - I believe on the next launch this would reset it to light mode.